EssentialFeed/EssentialFeediOS/ImageComments UI/Controllers/ImageCommentCellController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ImageCommentCellController.swift
+//  EssentialFeediOS
+//
+//  Created by Sebastian Vidrea on 03.04.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import EssentialFeed
+
+public final class ImageCommentCellController: NSObject {
+	private let viewModel: ImageCommentViewModel
+	private var cell: ImageCommentCell?
+
+	public init(viewModel: ImageCommentViewModel) {
+		self.viewModel = viewModel
+	}
+}
+
+extension ImageCommentCellController: UITableViewDataSource {
+	public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+		1
+	}
+
+	public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		cell = tableView.dequeueReusableCell()
+		cell?.messageLabel.text = viewModel.message
+		cell?.userNameLabel.text = viewModel.username
+		cell?.createdAtLabel.text = viewModel.createdAt
+		return cell!
```

Comment:
There's no need to hold a reference to the cell since the controller has access to the view model up front.

```suggestion
		let cell: ImageCommentCell = tableView.dequeueReusableCell()
		cell?.messageLabel.text = viewModel.message
		cell?.userNameLabel.text = viewModel.username
		cell?.createdAtLabel.text = viewModel.createdAt
		return cell
```